spec/spec_helper.rb

```diff
@@ -21,4 +30,14 @@
   config.define_cassette_placeholder(Rails.application.secrets.ibm_power_vc_defaults[:password]) do
     Rails.application.secrets.ibm_power_vc[:password]
   end
+
+  config.filter_sensitive_data('xxx') do |interaction|
```

Comment:
@agrare Do we have a standard substitution string that we use elsewhere (as opposed to "xxx")

Reply:
We typically use the provider name like `IBM_POWER_VC_AUTHORIZATION` so it is more obvious what is being filtered